conf/projects/toradex-verdin-am62/layers.conf

```diff
@@ -0,0 +1,30 @@
+BBPATH = "${TOPDIR}"
+BBFILES ?= ""
+
+BBLAYERS = "\
+    ${TOPDIR}/sources/meta-yoe \
+    \
+    ${TOPDIR}/sources/meta-toradex-bsp-common \
+    ${TOPDIR}/sources/meta-toradex-ti \
+    ${TOPDIR}/sources/meta-ti/meta-ti-bsp \
+    ${TOPDIR}/sources/meta-ti/meta-ti-extras \
+    ${TOPDIR}/sources/meta-arm/meta-arm \
+    ${TOPDIR}/sources/meta-arm/meta-arm-toolchain \
```

Comment:
do we need the arm toolchain layer ?

Reply:
Appears meta-ti-bsp depends on it:

`ERROR: Layer 'meta-ti-bsp' depends on layer 'meta-arm', but this layer is not enabled in your configuration`

Reply:
thats `meta-arm/meta-arm` which is fine, but I am not sure if we need `meta-arm/meta-arm-toolchain`

Reply:
from meta-arm layer.conf:

![image](https://github.com/YoeDistro/yoe-distro/assets/402813/da6a5404-77be-4fd1-8f4a-9b38c19515de)